examples/python/0_basic/cuda/basic_example_qwen3_fp8.py

```
@@ -0,0 +1,209 @@
+        print(f"  Decode time   : {decode_time*1000:.1f} ms  ({decode_tps:.1f} tok/s)")
+        print(f"  Output        : {output_text[:200]}")
+
+    # ── Summary ────────────────────────────────────────────────────────────────
+    avg_prefill_tps = total_prefill_tokens / total_prefill_time if total_prefill_time > 0 else 0
+    avg_decode_tps = total_decode_tokens / total_decode_time if total_decode_time > 0 else 0
+
+    print(f"\n{'='*60}")
+    print("[Qwen3-FP8] THROUGHPUT SUMMARY")
+    print(f"  Model           : {model_name}")
+    print(f"  GPUs            : {device_list}")
+    print(f"  FP8 blockwise   : enabled (e4m3fn, 128x128 blocks)")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)